### PR TITLE
Fix Compilation with Android NDK r26

### PR DESCRIPTION
**Pull request by BieHDC**
### Description:

Fixes Compilation with [Android NDK r26](https://github.com/android/ndk/wiki).
Does not break compatibility with r25c.
This is in preperation to add this path of building Android Applications as an option.

### Checklist:

- [ ] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

### CODE REVIEW

File: internal/driver/mobile/android.c [hunk 1]
```diff
@@ -178,12 +178,12 @@ char* readStream(uintptr_t jni_env, uintptr_t ctx, void* stream, int len, int* t
 		return NULL;
 	}
 
-	char* bytes = malloc(sizeof(char)*count);
+	jbyte* bytes = (jbyte*)malloc(sizeof(char)*count);
```
Comment (andydotxyz) [inline internal/driver/mobile/android.c:181]: Can you guarantee jbyte and char are the same size, or should the sizeof parameter be updated too?

**Review by andydotxyz — COMMENTED** [review]
Thanks for this - just a question inline

File: internal/driver/mobile/android.c (hunk 1 above)
Comment (BieHDC) [reply internal/driver/mobile/android.c]: Oversight. Fixed.

**Review by andydotxyz — APPROVED** [review]
Thanks so much

**Comment by andydotxyz** [comment]
Thanks so much, we appreciate your contribution.

**Pull request merged**